code/processes/filealerter.q

```diff
@@ -9,6 +9,7 @@ skipallonstart:@[value;`.fa.skipallonstart;0b]							// Whether to skip all acti
 moveonfail:@[value;`.fa.moveonfail;0b]								// If the processing of a file fails (by any action) then whether to move it or not regardless
 os:$[like[string .z.o;"w*"];`win;`lin]
 usemd5:@[value; `.fa.usemd5; 1b]								// Protected evaluation, returns value of usemd5 (from .fa namespace) or on fail, returns 1b
+processduplicate:@[value;`.fa.processduplicate;({x})]          // Run processduplicate from fa namespace on duplicate filenames passed in
```

Comment:
This needs to be a {{x}} (a function which returns a function) rather than ({x}) (a function)
Otherwise the default definition will be set to be the result of {x} with the error string applied, so probably something like ".fa.processduplicate"